src/core/api/index.js

```diff
@@ -62,27 +63,35 @@ export function callApi({
 
   return fetch(apiURL, options)
     .then((response) => {
-      return response.clone().json()
-        .then(
-          (jsonResponse) => ({ response, jsonResponse }),
-          (error) => {
+      function makeTextResponseJson(textResponse) {
+        return { jsonResponse: { text: textResponse }, response };
+      }
+
+      if (response.headers.get('Content-Type') === 'application/json') {
```

Comment:
It appears as though this will actually return a string and the "content-type" header is special.

The string does appear to be case-insensitive. Setting the header with `'content-type'` and retrieving it with `'Content-Type'` worked for me.